### PR TITLE
chore(Storybook): avatar, badge, callout, chip and dialog stories refactor

### PR DESCRIPTION
**Pull request by marcoskolodny**

### CODE REVIEW

File: src/__stories__/chip-story.tsx
Comment (yceballost) [inline src/__stories__/chip-story.tsx:1]: I think could be a good idea to have different stories for default / single selection / multiple selection
image

Comment (yceballost) [inline src/__stories__/chip-story.tsx:1]: Add a control to show / hide icon in single selection and multiple selection

File: src/__stories__/dialog-story.tsx
Comment (yceballost) [inline src/__stories__/dialog-story.tsx:1]: Same here.. I think is better use different story for different typologies of modals, IMO, these are different components and each has different controls 🤔

**Review by yceballost — DISMISSED** [review]

File: src/__stories__/dialog-story.tsx
Comment (marcoskolodny) [reply src/__stories__/dialog-story.tsx:1]: fixed

File: src/__stories__/chip-story.tsx
Comment (marcoskolodny) [reply src/__stories__/chip-story.tsx:1]: added

Comment (marcoskolodny) [reply src/__stories__/chip-story.tsx:1]: fixed

File: src/__stories__/badge-story.tsx [hunk 1]
```diff
@@ -1,60 +1,38 @@
...
+            options: ['undefined', '0', '2', '14'],
+            control: {type: 'select'},
+        },
+    },
+    parameters: {fullScreen: true},
 };
 
-export const Default: StoryComponent = () => {
+type Args = {
+    overInverse: boolean;
```
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/badge-story.tsx]: fixed

File: src/__stories__/callout-story.tsx [hunk 2]
```diff
@@ -55,28 +46,20 @@ export const Default: StoryComponent<Args> = ({
...
-                            button={button}
-                            secondaryButton={secondaryButton}
-                            buttonLink={buttonLink}
-                            aria-label="Callout label"
-                        />
-                    </Box>
-                </ResponsiveLayout>
-            </div>
-        </ThemeVariant>
+        <ResponsiveLayout isInverse={isOverInverse}>
```
Comment (yceballost) [inline src/__stories__/callout-story.tsx:49]: same here with inverse / isOverInverse, we try to keep consistency between stories

**Review by yceballost — DISMISSED** [review]

File: src/__stories__/callout-story.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/__stories__/callout-story.tsx]: fixed. It was also updated in all the button's stories

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**